corrscope/gui/__init__.py

```diff
@@ -1185,6 +1225,27 @@ def data(self, index: QModelIndex, role=Qt.DisplayRole) -> Any:
             if key == "wav_path" and role == Qt.DisplayRole:
                 if Path(value).parent != Path():
                     return "..." + Path(value).name
+
+            if key == "trigger_wav_path" and role == Qt.DisplayRole:
+                path = Path(value)
+                render_path = Path(self.channels[row].wav_path)
+
+                if path.name == render_path.name:
+                    # If name clashes with render_path, abbreviate past 2 components.
+                    if path.parent.parent != Path():
```

Comment:
- use `os.path.dirname` so `./foo/bar` doesn't fail to be abbreviated?
- `./foo/bar` is shorter than `...foo/bar`?
- nobody uses this kind of insane path?